podcast.rss

```diff
@@ -1021,5 +1022,132 @@
     </item>
     <!-- END: chapter 107 - maoon -->
     <!-- ==================================================================================== -->
+    <!-- BEGIN: chapter 106 -Quraysh -->
+ <item>
+  <title>92. സൂറ ഖുറൈഷ്  -പാരായണം</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #92
+💠Surah Quraysh 💠<br/>
+പാരായണം (Tajweed)💠</p>
+<p>🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/106-Quraysh /1-vqd-podcast-unit106.m4a" type="audio/m4a" length="1180000"/>
+      <itunes:duration>12:56</itunes:duration>
+      <pubDate>Friday, 03 May ‎2019 ‏‎12:00:00 GMT</pubDate>
+      <guid>1-vqd-podcast-unit106</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>93. സൂറ ഖുറൈഷ് -ചരിത്ര പശ്ചാത്തലം</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #93</p><p>💠Surah Quraysh 💠<br/>
+ചരിത്ര പശ്ചാത്തലം </p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/106-Quraysh /2-vqd-podcast-unit106.m4a" type="audio/m4a" length="1220000"/>
+      <itunes:duration>12:11</itunes:duration>
+      <pubDate>Friday, 03 May ‎2019 ‏‎12:00:00 GMT</pubDate>
+      <guid>2-vqd-podcast-unit106</guid>
+    </item>
+    <!--  -->
+    <item>
+    <title>94. സൂറ  ഖുറൈഷ് -ചരിത്ര പശ്ചാത്തലം </title>
```

Comment:
Please keep uniform spacing in the title. for e.g.,  `<title>94. സൂറ  ഖുറൈഷ് - ചരിത്ര പശ്ചാത്തലം </title>`

Reply:
Applied to all titles in this commit